Books-Labels/schema.sql

```diff
@@ -0,0 +1,24 @@
+CREATE DATABASE catalog;
+
+CREATE TABLE item (
+    id  int PRIMARY KEY,
+    publish_date DATE,
+    archived boolean,
+    FOREIGN KEY (label_id) REFERENCES labels (id)
+);
+
+CREATE TABLE books (
+    id integer PRIMARY KEY,
+    title varchar(100),
+    publisher varchar(100),
+    cover_state varchar(100),
+    publish_date,
+    archived boolean,
+    FOREIGN KEY(id) REFERENCES item(id)
+);
+
+CREATE TABLE labels (
```

Comment:
It would be better if the `labels` table was before the `item` table because the `item` table references the `labels` table and if we wanted to run all these commands at once we would get an error.